### DISCUSSION TITLE
OpenLDAP not available on Centos 8

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#39] [General]
OpenLDAP appears to have been deprecated in RHEL/Centos 7.4 and is no longer available on RHEL/Centos 8. 

Installation fails as Centos 8 cannot download and install openldap-server.

Suggest a note is added to the readme guide to warn of this limitation.

**Comment by dejo1307** [comment]
Hm, I've installed it just two hours ago on CentOS 7.9. I do believe (but like in April 2020) oldest CentOS I tested it on was 7.2, and didn't add any specific extra repos for that:

[base]
name=CentOS-$releasever - Base
mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
#baseurl=http://mirror.centos.org/centos/$releasever/os/$basearch/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

[root@centos79 yum.repos.d]# yum list installed | grep ldap
openldap.x86_64                            2.4.44-22.el7               @base
openldap-clients.x86_64                    2.4.44-22.el7               @base
openldap-devel.x86_64                      2.4.44-22.el7               @base
openldap-servers.x86_64                    2.4.44-22.el7               @base

[root@centos79 yum.repos.d]# cat /etc/centos-release
CentOS Linux release 7.9.2009 (Core)

Can you check if the mirror from above is enabled in your case?

And thanks, as always, for pointing this out :)

**Reply by michallawson** [reply]
From what I have read  it was deprecated at version 7.4 but was left in the later releases.
It was only removed from version 8 onwards. 
I overcame the problem by creating a version 7 server.
